pyp5js/python_functions.py

```diff
@@ -0,0 +1,9 @@
+class PythonFunctions:
```

Comment:
All of those 3 methods from `class` **_PythonFunctions_** are static, since the _self_ parameter is completely ignored.
So there's no actual need to have an instance of **_PythonFunctions_** in order to access them.
And to allow static access to those methods, we can re-write **_PythonFunctions_** like this, using built-in function **type()**:
```python
PythonFunctions = type('PythonFunctions', (), {
    'python_map': map,
    'python_filter': filter,
    'python_set': set
})
```

Reply:
Good suggestion, thanks!

Reply:
Hey @GoToLoop I have tried here and strangely I am getting an error `PythonFunctions.python_map is not a function`. It has worked when I imported that in ipython, but when I try to import to pyp5js it fails.

Reply:
@isabarros this can be breaking because of how `Transcrypt` implements the `type`. Eventually is still not possible to dynamically define classes on the fly with it. I've tested the following refactoring and it worked for me:

```
class PythonFunctions:

    @staticmethod
    def map(*args):
        return map(*args)

    @staticmethod
    def filter(*args):
        return filter(*args)

    @staticmethod
    def set(*args):
        return set(*args)
``` 

Then you can the methods with `PythonFunction.map` instead of accessing this `_PYTHON_INSTANCE`

Reply:
@isabarros this can be breaking because of how `Transcrypt` implements the `type`. Eventually is still not possible to dynamically define classes on the fly with it. I've tested the following refactoring and it worked for me:

```python
class PythonFunctions:

    @staticmethod
    def map(*args):
        return map(*args)

    @staticmethod
    def filter(*args):
        return filter(*args)

    @staticmethod
    def set(*args):
        return set(*args)
``` 

Then you can the methods with `PythonFunction.map` instead of accessing this `_PYTHON_INSTANCE`